### PR TITLE
create ui for login screen

### PR DESCRIPTION
**Pull request by fs-sournary**
### Summary:
- Login function

### Why need create this PR:
- create ui for login screen

### Effect:
- Login screen

### Link Issue:
screenshot_1527128346

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="dp_196">196dp</dimen>
+    <dimen name="dp_48">48dp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+</resources>
```
Comment (nguyentrungduc) [inline app/src/main/res/values/dimens.xml]: add blank line

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -10,17 +12,19 @@
...
-        <activity android:name=".screen.main.MainActivity">
+        <activity android:name=".screen.login.LoginActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN"/>
 
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
 
+        <activity android:name=".screen.main.MainActivity">
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: self close tag

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">FGoal</string>
+    <string name="hint_username">Username</string>
+    <string name="hint_password">Password</string>
+    <string name="label_login">Login</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: action login

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">FGoal</string>
+    <string name="hint_username">Username</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: User name

**Pull request merged**